test/e2e/404-page-router/index.test.ts

```diff
@@ -149,7 +149,10 @@ describe('404-page-router', () => {
           () => browser.eval('next.router.isReady ? "yes" : "no"'),
           'yes'
         )
-        expect(await browser.elementById('query').text()).toEqual('test=1')
+
+        await retry(async () => {
+          expect(await browser.elementById('query').text()).toEqual('test=1')
```

Comment:
Ideally this should use Playwright API like `await expect(page.locator('#query')).toHaveText('test=1');`